source/geh_calculated_measurements/src/geh_calculated_measurements/common/application/model/calculated_measurements_internal.py

```diff
@@ -45,5 +45,8 @@ def __init__(self, df: DataFrame):
             T.StructField(ContractColumnNames.observation_time, T.TimestampType(), not nullable),
             # The calculated quantity
             T.StructField(ContractColumnNames.quantity, T.DecimalType(18, 3), not nullable),
+            #
+            # Settlement type - "up_to_end_of_period" | "end_of_period"
```

Comment:
nit: Align with how we write the other contracts
```suggestion
            # Settlement type
            # "up_to_end_of_period" | "end_of_period"
```